BigQuery/14_final_schema.sql

```diff
@@ -0,0 +1,101 @@
+
+-- This schema is *SIMILAR* but not exactly the same as the PNC 2019 Challenge Schema
+-- Changing this to the correct schema should be very simple
+-- Please feel free to modify this yourself (add/rename/remove fields) to get it into the schema needed
+
+CREATE OR REPLACE TABLE `physionet.final_schema` as
+
+SELECT 
+subject_id,
+hadm_id,
+stay_id,
+chart_hour,
+admittime,
+deathtime,
+admission_type,
+first_careunit,
+last_careunit,
+total_los,
+ethnicity,
+gender,
+anchor_age as age,
+-- Cultures fields
+blood_cx,
+urine_cx,
+sputum_cx,
+-- Antibiotics fields (input events)
+amount as antibiotics_amount,
+rate as antibiotics_rate,
+--Antibiotics fields (pharmacy)
+abx_orders,
+-- vitals
+resp_rate,
+heart_rate,
+temperature,
+systolic_bp,
+diasystolic_bp as diastolic_bp,
+mean_arterial_bp,
+etco2,
+spo2,
+-- labs
+bicarbonate,
+chloride,
+bilirubin_indirect,
+bilirubin_direct,
+bilirubin_total,
+calcium,
+creatinine,
+glucose,
+lactate,
+magnesium,
+phosphate,
+potassium,
+troponin_I,
+hematocrit,
+hemoglobin,
+ptt,
+wbc,
+fibrinogen,
+platelets,
+-- ABG
+PaO2,
+SaO2,
+PaCO2,
+pH,
+base_excess,
+-- supplemental flow
+FiO2,
+O2_flow_rate,
+-- vent status
+ventilated
+FROM `elevated-pod-307118.physionet.demographic_and_stay_data`
+FULL JOIN
+`elevated-pod-307118.physionet.cultures_hourly`
+USING(subject_id, hadm_id, stay_id, chart_hour)
```

Comment:
I know, I'm repeating myself here, but I wanted to point out the repetition of using the subject id, hadm id, and stay id together when only one is needed.